scripts/java-api-check-all.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -Eeuo pipefail
+set -Exeuo pipefail
```

Comment:
Should it be a part of the public script? Looks more like a debug option for me

Reply:
It is indeed added to debug the logic - will remove it before merge